docs/test-pr-guide.md

```diff
@@ -10,7 +10,7 @@
 
 4. Go to the Issue that PR is fixing and follow the steps to reproduce that issue while you are under the develop branch.
 
-5. Run the following commands to get to the PR branch, where `<contributor>` is the GitHub username of the contributor that submitted the PR:
+5. Run the following commands to get to the PR branch, where `<id>` is the PR id and `<branch_name>` is the random name for the branch.
 
```

Comment:
Also add "Alternatively if you use the github cli you can run the following command `gh pr checkout <pr_ number>`"

Reply:
Do we need to use github cli now?

Reply:
We don't need to it's just an alternative to this. I don't see why we'd need to change the branch name while reviewing a pr 😕.

Reply:
Most of the times, new Contributor made PR from branch named `develop` and while trying to fetch locally it cause errors. Anyways, it's my personal opinion that we should change branch name .

Reply:
> Most of the times, new Contributor made PR from branch named `develop` and while trying to fetch locally it cause errors. Anyways, it's my personal opinion that we should change branch name . 

Sorry I still don't get it. The branch name would be whatever the contributor chooses right? I don't see how it affects the testing.